dodo.py

```diff
@@ -425,29 +425,32 @@ def task_dev():
             "--no-index",
             "--find-links",
             B.DIST,
+            py_name,
         ]
-        file_dep = []
-        for py_name in [C.NAME, C.CORE_NAME]:
-            py_name_pkg = py_name.replace("-", "_")
-            file_dep += [B.DIST / f"""{py_name_pkg}-{D.PY_VERSION}-{C.NOARCH_WHL}"""]
-            args += [py_name]
 
         yield dict(
             name="py:jupyterlite-core",
             actions=[U.do(*args, cwd=P.ROOT)],
-            file_dep=file_dep,
+            file_dep=[B.DIST / f"""{py_name}-{D.PY_VERSION}-{C.NOARCH_WHL}"""],
         )
     else:
-        for py_name in [C.NAME, C.CORE_NAME]:
-            py_name_pkg = py_name.replace("-", "_")
-            cwd = P.PY_SETUP_PY[py_name].parent
-            file_dep = [cwd / "src" / py_name_pkg / B.APP_PACK.name]
-            args = [*C.FLIT, "install", "--pth-file", "--deps=none"]
+        core_args = [*C.PYM, "pip", "install", "-e", "./py/jupyterlite-core[test]"]
+        yield dict(
+            name="py:jupyterlite-core",
+            actions=[U.do(*core_args, cwd=P.ROOT)],
+        )
 
+        metapackage_args = [
+            *C.PYM,
+            "pip",
+            "install",
+            "-e",
+            "./py/jupyterlite",
+            "--no-deps",
+        ]
         yield dict(
-            name=f"py:{py_name}",
-            actions=[U.do(*args, cwd=cwd)],
-            file_dep=file_dep,
+            name="py:jupyterlite",
+            actions=[U.do(*metapackage_args, cwd=P.ROOT)],
```

Comment:
removing and existing `file_dep` is a recipe for having to re-run stuff over and over again with no reason, and probalby out of order.

Reply:
yeah probably we can add it back. This is likely a leftover of fighting with the doit tasks

Reply:
Actually I don't know, this task is just here so we can simply install the packages in editable mode...

Reply:
Right: but it would _at least_ depend on:
- `pyproject.toml` (changing `entry_points`, `data_files`, and a raft of other things invalidates an editable install)
- anything indirectly consumed _by_ `pyproject.toml` (e.g. `_version.py` or wherever the source of truth is these days)

Reply:
I'm planning to have another look at this as part of https://github.com/jupyterlite/jupyterlite/issues/684